shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2513,6 +2537,15 @@ - (void)takeKeyboardScreenshotAndDisplay {
   _keyboardViewContainer.frame = _keyboardRect;
 }
 
+- (void)showNativeEditMenu:(NSDictionary*)dictionary {
+  if (@available(iOS 16.0, *)) {
+    // TODO: get point and arrowDirection from args
+    CGPoint point = CGPointMake(100, 100);
+    UIEditMenuArrowDirection arrowDirection = UIEditMenuArrowDirectionUp;
```

Comment:
@justinmc For Milestone 1, I just need a `point` and `arrowDirection` from the framework. 

Note that for arrow direction, `up` actually means the edit menu is BELOW the text field (triangle pointing upwards)

Reply:
I believe global coordinates is the convention. I just checked and the Scribble feature sends global Offsets in its method channel calls.

The way that arrow position works in the framework is that it tries to fit above the anchor point, but if it's too big, then it goes below. So for the framework to provide arrowDirection are we able to hardcode the height of the native menu in the framework?

Also, if you for example purposely position the menu too close to the top and use arrowDirection `down`, does it hang off the screen?

Reply:
> Also, if you for example purposely position the menu too close to the top and use arrowDirection down, does it hang off the screen?

When the menu is positioned exceeding the bound, then it will be capped within bound. For example, below I set `y` to be `-1000`: 

![Simulator Screenshot - iPhone 15 Pro Max - 2024-01-26 at 13 10 29](https://github.com/flutter/engine/assets/41930132/43763b39-9188-4760-9124-d46ede454f70)


> The way that arrow position works in the framework is that it tries to fit above the anchor point, but if it's too big, then it goes below. So for the framework to provide arrowDirection are we able to hardcode the height of the native menu in the framework?

Interesting. Let me try to see if it is possible to do similar in the engine (e.g. get the size of the menu). If it's possible, then the framework can send 2 points (top and bottom), and engine can decide which one to use; 
If not possible, then we will have to use a "magic number" (either engine side or framework side)

Reply:
Sounds good. On the framework we typically already have both an `anchorAbove` and `anchorBelow`, so it would be easy to send both.

Reply:
On the framework do you have the rectangle area? I found the new API that takes a rect and iOS will decide the arrow direction automatically: https://github.com/flutter/engine/pull/50095#discussion_r1468211520